README.md

```diff
@@ -1,2 +1,53 @@
 # kubejob
-Run Kubernetes jobs and wait for their completion
+Run Kubernetes jobs and wait for their completion.
+
+## Motivation
+Sometimes you just want to run a job in a Kubernetes cluster and wait for its completion while
+keeping an eye on the container output. This usecase frequently appears in deployment pipelines
+eg. for running a smoke test or a database migration before the actual deployment continues. 
+
+## How to install
+```bash
+go get -v -u github.com/egymgmbh/kubejob
+```
+
+## How to use
```

Comment:
I think the backoff limit in the job configuration is still quite important correct? Could we maybe add it here or comment it in the .yaml? Or what happens when the backofflimit is greater than 0?

Reply:
added documentation and a warning if not set correctly